lib/config/config-file.js

```diff
@@ -395,6 +396,14 @@ function load(filePath, applyEnvironments) {
             Plugins.loadAll(config.plugins);
         }
 
+        // include full path of parser if present
+        if (config.parser) {
+            config.parser = resolveModule.sync(config.parser, {
+                basedir: path.dirname(filePath),
```

Comment:
Though I'm not sure it's a problem here, but I just remembered https://github.com/substack/node-resolve/issues/78